src/pykeen/cli.py

```diff
@@ -372,19 +372,21 @@ def metrics(tablefmt: str):
 
 
 def _help_metrics(tablefmt):
+    headers = [
+        "Name",
+        "Interval",
+        "Direction",
+        "Description",
+        "Type",
+        "Closed-Form Expectation",
```

Comment:
do these check boxes really add important information for the readme? They make the formatting really bad so I'd say we should omit them

Reply:
678cc8f